<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <addaction name="menuReservas"/>
   <addaction name="menuClientes"/>
   <addaction name="menuVehiculos"/>
   <addaction name="menuAdministracion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <action name="menuReservas">
  <property name="text">
   <string>Reservas</string>
  </property>
 </action>
 <action name="menuClientes">
  <property name="text">
   <string>Clientes</string>
  </property>
 </action>
 <action name="menuVehiculos">
  <property name="text">
   <string>Vehículos</string>
  </property>
 </action>
 <action name="menuAdministracion">
  <property name="text">
   <string>Administración</string>
  </property>
 </action>
 <resources/>
 <connections/>
</ui>
